File: app/src/main/java/com/example/javakotlin/javakotlinoverview/java/ErrorHandlingThrow.java
package com.example.javakotlin.javakotlinoverview.java;

public class ErrorHandlingThrow {
    static void checkPassword(String password) {
        if (password.length() < 8) {
            throw new ArithmeticException("Password too short! Try again");
        }
        else {
            System.out.println("Access granted!");
        }
    }

    public static void main(String[] args) {
        checkPassword("bansi");
    }
}

File: app/src/main/java/com/example/javakotlin/javakotlinoverview/kotlin/ErrorHandlingThrow.kt
package com.example.javakotlin.javakotlinoverview.kotlin

class ErrorHandlingThrow {
    fun main(args: Array<String>) {
        checkPassword("bansi")
    }
    fun checkPassword(password: String) {
        if (password.length < 6)
            throw ArithmeticException("Password is too short")
        else
            println("Strong password")
    }
}